src/charm.py

```diff
@@ -173,7 +185,9 @@ def _create_env(self):
             "SUPERSET_SECRET_KEY": self._state.superset_secret,
             "ADMIN_PASSWORD": self._state.admin_password,
             "CHARM_FUNCTION": self.config["charm-function"],
+            "SQL_ALCHEMY_URI": self._state.sql_alchemy_uri,
         }
+        logging.info(env)
```

Comment:
I don' think it is a good idea to log something that contains secrets.

Reply:
You are completely right, this was just in while I was testing. Good spot, removed.